test-cf-stack.json

```diff
@@ -87,6 +87,55 @@
       "DependsOn": [
         "DockerRunnerUser"
       ]
+    },
+    "DockerRunnerKMSKey": {
+      "Type": "AWS::KMS::Key",
+      "Properties": {
+        "Description" : "Docker Runner Test Key",
+        "KeyPolicy": {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Sid": "Enable IAM User Permissions",
+              "Effect": "Allow",
+              "Principal": {
+                "AWS": "arn:aws:iam::061240556736:root"
+              },
+              "Action": "kms:*",
+              "Resource": "*"
+            },
+            {
+              "Sid": "Allow use of the key",
+              "Effect": "Allow",
+              "Principal": {
+                "AWS": {
+                  "Fn::Join": ["", ["arn:aws:iam::061240556736:user/", {"Ref": "DockerRunnerUser"}]]
```

Comment:
jak tohle budem nasazovat do produkce / testingu? nebylo by jednodušší CF template pro KMS udělat úplně bokem v `keboola/object-encryption`?

Reply:
tohle je jen pro testy 

nasazeni: https://keboola.slack.com/archives/C31U6BGJC/p1513157792000047?thread_ts=1513122326.000365&cid=C31U6BGJC

do keboola/object-encryption bych to nepletl, tomu je potom prd, kde ty klice sezenes